packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -117,7 +192,7 @@ export class VaTelephone {
       </Fragment>
     ) : (
       <a
-        href={VaTelephone.createHref(contact, extension)}
+        href={VaTelephone.createHref(contact, extension, this.alphaPhoneNumber)}
```

Comment:
The test shows that `VETS` isn't included in the href, but it looks like we may be putting it in anyway. This makes me think that the browser is stripping out invalid characters from a `tel` link which is great, but let's not trust all browsers to do that. We should probably leave the `href` alone.

Reply:
@bkjohnson the href if left alone becomes href=tel:+1877222VETS which isn't a valid phone number which is why we also need to convert it in the createHref as well from letters to numbers